app/pages/resource/info/ResourceInfo.spec.js

```diff
@@ -20,13 +23,42 @@ describe('pages/resource/info/ResourceInfo', () => {
     return shallow(<ResourceInfo {...defaults} {...props} />);
   }
 
-  it('renders unit name', () => {
-    const unitName = getWrapper().find('.unit-name');
-    expect(unitName.text()).to.equal(unit.name.fi);
+  describe('header', () => {
+    let header;
+    before(() => {
+      header = getWrapper().find('header');
+    });
+
+    it('renders unit name', () => {
+      const unitName = header.find('.unit-name');
+      expect(unitName.text()).to.equal(unit.name.fi);
+    });
+
+    it('renders resource name', () => {
+      const resourceName = header.find('.resource-name');
+      expect(resourceName.text()).to.equal(resource.name.fi);
+    });
+
+    it('renders resource address', () => {
+      const unitAddress = header.find('.unit-address');
+      expect(unitAddress.text()).to.equal(unit.streetAddress.fi);
+    });
   });
+  describe('resource-details section', () => {
+    let section;
+    before(() => {
+      section = getWrapper().find('section.resource-details');
+    });
+
+    it('renders resource type', () => {
+      const resourceType = section.find('.resource-type');
+      expect(resourceType.text()).to.equal(resource.type.name.fi);
+    });
 
-  it('renders resource name', () => {
-    const resourceName = getWrapper().find('.resource-name');
-    expect(resourceName.text()).to.equal(resource.name.fi);
+    it('renders resource people capacity', () => {
+      const resourceCapacity = section.find('.resource-people-capacity');
+      expect(resourceCapacity.children().get(0).props.children).to.equal('Henkilömäärä: ');
```

Comment:
- [ ] These expects could pretty easily break if there are any changes in the html structure of the 'div.resource-people-capacity'. I would rather use for exmple:
```javascript
expect(resourceCapacity.text()).to.equal(`Henkilömäärä: ${resource.peopleCapacity}`);
```